src/gui/ApplicationSettingsWidgetSecurity.ui

```diff
@@ -168,7 +168,17 @@
       <item>
        <widget class="QCheckBox" name="quickUnlockCheckBox">
         <property name="text">
-         <string>Enable database quick unlock (Touch ID / Windows Hello)</string>
+         <string>Enable database quick unlock by default</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QCheckBox" name="quickUnlockRememberCheckBox">
+        <property name="toolTip">
+         <string>Quick unlock can only be remembered when using Touch ID or Windows Hello</string>
+        </property>
+        <property name="text">
+         <string>Remember quick unlock after database is closed (Touch ID / Windows Hello only)</string>
         </property>
```

Comment:
Remove the part in parentheses and show this option conditionally only when either of the two is available.